### PR TITLE
open-api updated

### PR DESCRIPTION
**Pull request by ahadvl**
Add /surah routers to open-api file.

### CODE REVIEW

File: open-api.json [hunk 1]
```diff
@@ -948,14 +1031,84 @@
...
+                        }
+                    }
+                },
                 "tags": [
                     "quran"
                 ],
                 "responses": {
                     "200": {
-                        "description": "Returns the single surah"
+                        "description": "This means new surah was edited"
```
Comment (nyzd) [inline open-api.json]: Maybe we can use existing or available surah instead of new?

**Review by nyzd — CHANGES_REQUESTED** [review]
Single comment.

File: open-api.json (hunk 1 above)
Comment (al-abd) [reply open-api.json]: fix this one and we merge pr, ty.

**Review by nyzd — APPROVED** [review]
Thanks for you contribution.

File: open-api.json (hunk 1 above)
Comment (Al-Abd) [reply open-api.json]: Just write: "This means surah was edited"

**Pull request merged**